core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/SchemaRegistryCoordinator.java

```diff
@@ -200,8 +232,15 @@ protected Map<String, ByteBuffer> onLeaderElected(
       error = SchemaRegistryProtocol.Assignment.DUPLICATE_URLS;
     }
 
-    // All members currently receive the same assignment information since it is just the leader ID
     Map<String, ByteBuffer> groupAssignment = new HashMap<>();
+    if (existingLeaderKafkaId != null) {
+      leaderKafkaId = existingLeaderKafkaId;
+      leaderIdentity = existingLeaderIdentity;
+      if (!identity.equals(leaderIdentity) && identity.isLeader()) {
```

Comment:
For my understanding. Does identity refer to the newly elected leader?

Reply:
identity refers to the current node's identity which may or may not be the leader.